share/shutter/resources/modules/Shutter/Pixbuf/Save.pm

```diff
@@ -49,17 +49,41 @@ sub new {
 	my $current_window = $self->{_window} || $self->{_common}->get_mainwindow;
 	$self->{_dialogs} = Shutter::App::SimpleDialogs->new($current_window);
 	$self->{_lp}      = Shutter::Pixbuf::Load->new($self->{_common}, $current_window);
+	$self->{_quality} = undef;
 
 	bless $self, $class;
 	return $self;
 }
 
+sub set_quality_setting {
+		my $self = shift;
+		my $filetype = shift;
```

Comment:
This accepts two parameters

Reply:
Sorry, got it, I should call it with `$self->set_quality_setting`, not just `set_quality_setting`...